stopcovid/status/reminder_triggers.py

```diff
@@ -0,0 +1,103 @@
+import uuid
+from dataclasses import dataclass
+from typing import List
+
+from sqlalchemy import (
+    Table,
+    MetaData,
+    Column,
+    String,
+    select,
+    exists,
+    Index,
+    and_,
+    insert,
+)
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import UniqueConstraint
+from sqlalchemy.exc import DatabaseError, IntegrityError
+
+from . import db
+
+metadata = MetaData()
+reminder_triggers = Table(
+    "reminder_triggers",
+    metadata,
+    Column("id", UUID, primary_key=True),
+    Column("drill_instance_id", UUID, nullable=False, index=True),
+    Column("prompt_slug", String, nullable=False),
+    # Do not remind for same prompt on drill twice
+    UniqueConstraint("drill_instance_id", "prompt_slug"),
+)
+
+
+Index(
+    "drill_instance_id_prompt_slug_index", "drill_instance_id", "prompt_slug",
+)
```

Comment:
I believe that this is redundant with the uniqueness constraint.

Reply:
and even it if isn't, it needs to be within the Table definition or it won't get created.